services/galley/test/integration/API/Util.hs

```diff
@@ -427,6 +427,7 @@ createTeamConvLegacy u tid us name = do
   let convPayload =
         object
           [ "users" .= us,
+            "qualified_users" .= ([] :: [Qualified UserId]),
```

Comment:
for backwards compatibility, it must be possible to not send qualified_users here. If this line is required for tests to pass, then the fromJSON instance of NewConv is wrong and needs to default to an empty list if not provided.